sql/moz-fx-data-shared-prod/telemetry_derived/experiments_daily_active_clients_v1/query.sql

```diff
@@ -1,12 +1,106 @@
+WITH desktop AS (
+  SELECT DISTINCT
+    submission_date,
+    e.key AS experiment_id,
+    e.value AS branch,
+    client_id
+  FROM
+    telemetry.clients_daily
+  CROSS JOIN
+    UNNEST(experiments) AS e
+),
+fenix AS (
+  SELECT DISTINCT
+    DATE(submission_timestamp) AS submission_date,
+    e.key AS experiment_id,
+    e.value.branch AS branch,
+    client_info.client_id
+  FROM
+    `moz-fx-data-shared-prod.org_mozilla_fenix_stable.metrics_v1`
```

Comment:
Let's prefer baseline pings here since they're lower-latency and less likely to be dropped